examples/applications/svm_gui.py

```diff
@@ -319,6 +319,7 @@ def get_parser():
 
 def main(argv):
     op = get_parser()
+
```

Comment:
Any reason to modify (or not modify) this file?

It looks like this example is using argparse but is also using a Tk GUI so I am not sure what is going to happen in a Jupyter notebook or in an IPython console.